include/swift/Driver/Action.h

```diff
@@ -328,16 +328,19 @@ class GeneratePCHJobAction : public JobAction {
 class DynamicLinkJobAction : public JobAction {
   virtual void anchor();
   LinkKind Kind;
+  bool LTO;
 
 public:
-  DynamicLinkJobAction(ArrayRef<const Action *> Inputs, LinkKind K)
+  DynamicLinkJobAction(ArrayRef<const Action *> Inputs, LinkKind K, bool LTO)
       : JobAction(Action::Kind::DynamicLinkJob, Inputs, file_types::TY_Image),
-        Kind(K) {
+        Kind(K), LTO(LTO) {
     assert(Kind != LinkKind::None && Kind != LinkKind::StaticLibrary);
   }
 
   LinkKind getKind() const { return Kind; }
 
+  bool PerformLTO() const { return LTO; }
```

Comment:
Please follow naming guidelines!

-> shouldPerformLTO().

Can you rename the flag to ShouldPerformLTO? What does bool LTO even mean? It is ambiguous.